### ISSUE TITLE
Custom Matches cannot have teams without team killing 

### ISSUE THREAD

**Issue by tara81** [BattlePlugins/BattleArena#16]
Teams can team kill with custom games.  I cannot seem to replicate in non custom games.
I cannot have teams in a custom game.

**Comment by Zach443** [comment]
Are you saying you can't have teams in a custom game because the plugin won't let you, or because it would be messed up with people being able to team kill?

**Comment by tara81** [comment]
Its messed up because people are able to team kill. I thought it was custom matches causing this, But I am wrong.  It is actually when you give teams a class in the config like this : 
  onSpawn:
    options:
    - hunger=20
    - deEnchant
    - respawnWithClass
    giveClass:
      team1: rebel
      team2: empire

**Comment by Zach443** [comment]
@Europia79 and I just tested this, and when we used a default arena team killing was not allowed, however when we used your config/ a custom game it was. We are looking into the cause of this, stay tuned for more info.

**Comment by Europia79** [comment]
@tara81 

Here's a temporary work-around:
http://rainbowcraft.sytes.net/download/plugins/StarWars/

it's your SWConfig.yml compiled into a BattleArena extension JAR. There are three of them: java8, java7, and java6.

It's not your exact SWConfig.yml... so you'll have to tweak it to your liking. Lemme know if this works.

**Comment by tara81** [comment]
thanks guys for looking into this, much appreciated =)

I thought I tested a default arena with the config but i guess I did something wrong.

I would pick java8 if java8 is installed on my server computer, correct?

After i put the jar on the server, what do I do then ? Create a new config within the ArenaStarWars folder? A little confused on how this works :)

**Comment by tara81** [comment]
Also would I then use /sw create arena or ?

**Comment by tara81** [comment]
@ Europia79 , Nope that didnt fix it =(

**Comment by Europia79** [comment]
@tara81 

i actually made a typo... Sorry about that... re-download it... make sure you have version 1.1

Put ArenaStarWars.jar in your /plugins/ folder

Also, make sure you delete SWConfig.yml in

```/plugins/BattleArena/competitions/SWConfig.yml```

And instead, use

```/plugins/ArenaStarWars/StarWarsConfig.yml```

to make changes. Hopefully it works this time. But unfortunately, I don't have a 2nd person to test this out. You can always message me on Skype too and i'll help you test to see if it works.

My Skype name is Nicodemis79

**Comment by Europia79** [comment]
Okay, me and Zach just tested this... This works as a valid work-around for now.

**Comment by tara81** [comment]
okay thanks a lot for fixing your typo !

**Comment by tara81** [comment]
So your fix worked this time. I however noticed that snd was way more fun for star wars minigame. (wow I only noticed these minigames existed, their awesome)

Is there a way to create custom matches with snd and bomb arena ?

**Comment by Europia79** [comment]
oh wow, Search-N-Destroy == the Rebels VS the Empire... awesome idea !

**Comment by lDucks** [comment]
Seems like a fix/work around was found? Is there an accepted resolution? @clovisd @Europia79

**Comment by Europia79** [comment]
I did just a work-around... not a properly implemented solution for all custom games.

I'm not sure exactly what the deal is... I tried to debug the code... But couldn't tell exactly what the problem was. I did notice that these custom games are handled differently than JAR extensions tho. Which seems to suggest that a solution would be to handle YML custom games and JAR custom games the same way: ```BattleArena.registerCompetition()```
